Add "wrap" to exports.

cahrens opened the pull request:
@stoeffel Here is my PR to fix the exports function.

stoeffel commented on tests/exports.js:
```diff
@@ -0,0 +1,9 @@
+var _ = require('underscore');
+var deepEqual = require('assert').deepEqual;
+var s = require('../dist/underscore.string');
```
could you change this to 
```diff
- var s = require('../dist/underscore.string');
+ var s = require('../');
```

stoeffel commented:
btw. to run the tests locally you can just run `npm test` in your shell.

cahrens commented:
@stoeffel I made the change you requested (also changed a usage of double quotes to single quotes, as it appears that you are standardizing on single quotes) and squashed the commit. Thanks for pushing out a release so quickly!

I took a look at #481 and #482, but don't really have the background to review them.

stoeffel commented:
Thanks for the PR.

> I took a look at #481 and #482, but don't really have the background to review them.

No problem. Thanks anyway.

stoeffel commented:
finally published `3.3.4`. had some troubles with the new build chain, but now it works fine. 
// cc @epeli
